pypicloud/views/packages.py

```diff
@@ -0,0 +1,21 @@
+""" View for cleaner buildout calls """
+from pyramid.view import view_config
+from pyramid_duh import addslash
+
+from pypicloud.route import PackagesResource
+from pypicloud.views.simple import _packages_to_dict
```

Comment:
Small nit: since this function is no longer local to the file where it's declared it should be renamed to `packages_to_dict`